mastodon/table_mastodon_account.go

```diff
@@ -123,7 +123,12 @@ func accountServerFromAccount(ctx context.Context, input *transform.TransformDat
 	account := input.Value.(*mastodon.Account)
 	re := regexp.MustCompile(`https://(.+)/`)
 	matches := re.FindStringSubmatch(account.URL)
-	return matches[1], nil
+	if len(matches) > 0 {
+		return matches[1], nil
+	} else {
+		plugin.Logger(ctx).Debug("accountServerFromAccount fail", "account", account, "url", account.URL)
+		return "unknown", nil
```

Comment:
@judell should this `return nil, nil`?

Is there a specific reason why we are returning `unknown`?

Also, do we get an error when we query an account with no URL?